cmd/calyptia/create_trace_session.go

```diff
@@ -90,15 +90,29 @@ func (config *config) completePipelinePlugins(pipelineKey string, cmd *cobra.Com
 		return nil, cobra.ShellCompDirectiveError
 	}
 
-	cfg, err := parsePipelineConfig(pipeline.Config)
+	conf, err := fluentbitconfig.ParseAs(pipeline.Config.RawConfig, fluentbitconfig.Format(pipeline.Config.ConfigFormat))
 	if err != nil {
 		return nil, cobra.ShellCompDirectiveError
 	}
 
+	// TODO: use instance id instead of name.
```

Comment:
This might be a breaking change, but only affects the autocompletion, its not validation or anything too bad.
Left it as TODO for now.